### DISCUSSION TITLE
CRT throwing up some odd characters when playing Infocom games

### DISCUSSION THREAD

**Discussion by oilcan-productions** [thehighnibble/thehighnibble.github.io#47] [IMSAI8080esp replica]
I have updated to the March release and did not notice that before but CRT shows escape characters.
![CRT](https://user-images.githubusercontent.com/77457397/232851833-8f81e7be-77b4-4439-83ba-16985f214c5d.png)

**Comment by udo-munk** [comment]
This looks like you try to run the software on the IMSAI VIO, which is not VT-100 compatible. The Infocom games all are patched for VT-100, so run them on the TTY.
